code/lib/cli/src/initiate.ts

```diff
@@ -313,22 +314,30 @@ async function doInitiate(options: CommandOptions, pkg: PackageJson): Promise<vo
   }
   done();
 
-  const storybookInstalled = isStorybookInstalled(packageJson, options.force);
+  const storybookInstatiated = isStorybookInstatiated(resolve(process.cwd(), '.storybook'));
```

Comment:
I think we should keep using the word installed 🤔

Reply:
Storybook "installed" can have multiple meanings. It can, for example, mean that Storybook is listed in the `package.json`.  `storybookInstalled` is a vague definition of the state and is, in my point of view misleading.

Reply:
Got it! thanks for explaining!